Cargo.toml

```diff
@@ -24,12 +24,16 @@ test = false
 doctest = false
 
 [features]
-default = ["spans", "std"]
+default = ["spans", "std", 'strict-macro']
```

Comment:
Note that this is on-by-default, but I don't expect to land it like this. I suspect we'll have to turn this to off-by-default (and perhaps on-by-deafult for the next major release). In the meantime this is being used to test CI.

Reply:
Since the other strings in this file use double quotes, this new default feature string should also use double quotes.